cputemp/app.go

```diff
@@ -20,29 +21,29 @@ func getTemperature() float64 {
 	temp := out.String()
 
 	//temp=35.8'C
-	tempVal := temp[strings.Index(temp, "=") + 1 : len(temp) - 3 ]
+	tempVal := temp[strings.Index(temp, "=")+1 : len(temp)-3]
```

Comment:
I use VSCode mostly which uses `gofmt` in this case it might have been me that added the extra spacing by using vim.